runtime/development/src/lib.rs

```diff
@@ -1302,13 +1371,13 @@ impl pallet_loans_ref::Config for Runtime {
 	type InterestAccrual = InterestAccrual;
 	type ItemId = ItemId;
 	type LoanId = LoanId;
-	type MaxActiveLoansPerPool = MaxActiveLoansPerPool;
+	type MaxActiveLoansPerPool = MaxActiveLoansPerPoolBenchmark;
 	type MaxWriteOffPolicySize = MaxWriteOffPolicySize;
 	type NonFungible = Uniques;
 	type Permissions = Permissions;
 	type Pool = PoolSystem;
-	type PriceId = PriceId;
-	type PriceRegistry = pallet_loans_ref::util::NoPriceRegistry<Runtime>;
+	type PriceId = OracleKey;
```

Comment:
I think this is a breaking change if we do it in `altair` or `centrifuge`. Do we need a migration for it, or are we still in the same state as the previous release, where it's still not used?

Reply:
I would think it doesn't require a migration if there are no loans with external pricing, right?

Reply:
I think so, it all depends on that